src/styles/utils/transition.css

```diff
@@ -0,0 +1,22 @@
+.item-enter {
+  opacity: 0;
+  height: 0;
+}
+
+.item-enter-active {
+  opacity: 1;
+  transition: all 400ms ease;
```

Comment:
It's generally a good practice to specify which properties you want to transition instead of using 'all'. This can prevent unwanted side-effects.